telegram-bot/src/main/java/com/exadel/telegrambot/bot/utils/Constant.java

```diff
@@ -0,0 +1,12 @@
+package com.exadel.telegrambot.bot.utils;
+
+public interface Constant {
```

Comment:
better use class for Constants instead of interface

Reply:
why I think interface is better, because we always user interface for constans

Reply:
Lets change it